src/pages/gallery/index.tsx

```diff
@@ -777,6 +778,10 @@ export default function Gallery() {
                                 activeCollection,
                                 collections
                             )}
+                            isUncategorizedCollection={isUncategorizedCollection(
```

Comment:
we should user collectionSummary to determine is an album is uncategorized.

That way we wouldn't have to loop through all collections

Reply:
also can you please update the above isFavoriteCollection to do so.

Reply:
also there is isSharedCollection, can you also update